homeassistant/components/surepetcare/sensor.py

```diff
@@ -67,14 +61,25 @@ def __init__(self, _id: int, spc: SurePetcareAPI) -> None:
         self._attr_unique_id = (
             f"{surepy_entity.household_id}-{surepy_entity.id}-battery"
         )
+        self._update_attr()
+
+    @property
+    def available(self) -> bool:
+        """Return True if entity is available."""
+        return super().available and bool(self.state)
```

Comment:
```suggestion
```

Usually we only set `available` based on if we cannot communicate with the api or the service cannot communicate with the device. 

Does the `available` provided by `CoordinatorEntity` need to be overwritten here?